server/apps/archive/archive_crop.py

```diff
@@ -98,22 +96,27 @@ def _validate_aspect_ratio(self, crop, doc):
         if doc_ratio != spec_ratio:
             raise SuperdeskApiError.badRequestError(message='Wrong aspect ratio!')
 
-    def _get_crop_by_name(self, crops, crop_name):
+    def get_crop_by_name(self, crop_name):
         """
         Finds the crop in the list of crops by name
-        :param crops: List of crops
         :param crop_name: Crop name
         :return: Matching crop or None
         """
+        crops = get_resource_service('vocabularies').find_one(req=None, _id='crop_sizes').get('items')
+        if not crops:
+            raise SuperdeskApiError.badRequestError(message='Crops couldn\'t be loaded!')
+
         return next((c for c in crops if c.get('name', '').lower() == crop_name.lower()), None)
 
     def _delete_crop(self, original, crop_name):
```

Comment:
IMO this is fine as it is already, but of course depends on whether this method is meant to be publicly used or is it just an internal helper method of the class.